execution_engine_testing/tests/src/test/system_contracts/auction/bids.rs

```diff
@@ -3068,3 +3082,999 @@ fn should_fail_with_more_accounts_than_slots() {
 fn should_run_genesis_with_exact_validator_slots() {
     check_validator_slots_for_accounts(DEFAULT_EXEC_CONFIG.validator_slots() as usize);
 }
+
+#[ignore]
+#[test]
+fn should_delegate_and_redelegate() {
+    let system_fund_request = ExecuteRequestBuilder::standard(
+        *DEFAULT_ACCOUNT_ADDR,
+        CONTRACT_TRANSFER_TO_ACCOUNT,
+        runtime_args! {
+            ARG_TARGET => *SYSTEM_ADDR,
```

Comment:
Why not have the system account funded by default in tests. We do this in almost every single unit test.